phoenix-core/src/it/java/org/apache/phoenix/end2end/index/MutableIndexFailureIT.java

```diff
@@ -385,6 +383,8 @@ private void updateTable(Connection conn, String tableName) throws SQLException
     }
 
     public static class FailingRegionObserver extends SimpleRegionObserver {
+        public static volatile boolean FAIL_WRITE = false;
```

Comment:
Sharing these may make the tests flaky if both MutableIndexFailureIT and IndexToolForPartialBuildIT run parallel.